apps/dashboard/hooks/use-hibiscus-user/use-hibiscus-user.ts

```diff
@@ -14,11 +17,44 @@ export interface UseHibiscusUser {
 }
 
 export function useHibiscusUser(): UseHibiscusUser {
-  const [user, setUser] = useState<HibiscusUser>({
-    tag: '@vincentvu',
-    role: HibiscusRole.ADMIN,
-    firstName: 'Vincent',
-  });
+  const [user, setUser] = useState<HibiscusUser>(null);
+
+  async function getUserProfile() {
+    // Get user profile from db
+    const supabase = container.resolve(HibiscusSupabaseClient);
+    const profile = await supabase.getUserProfile(
+      getCookie(process.env.NEXT_PUBLIC_HIBISCUS_ACCESS_COOKIE_NAME) as string,
+      getCookie(process.env.NEXT_PUBLIC_HIBISCUS_REFRESH_COOKIE_NAME) as string
```

Comment:
nit: try using and adding stuff from `@hibiscus/env`

Reply:
Done!